pre-processing/primod/driver_coupling/ribamod.py

```diff
@@ -106,6 +106,27 @@ def write_exchanges(
                     save_flows=True,
                 )
                 destination = "river"
+
+                #  check on the bottom elevation and ribasim minimal subgrid level
+                minsglevel = (
+                    ribasim_model.basin.subgrid.df.groupby("subgrid_id")
+                    .min()[["subgrid_level"]]
+                    .to_numpy()
+                    .ravel()
+                )
```

Comment:
this would give the same results right: 

1: [["subgrid_level"]] -> ["subgrid_level"]
2: remove ravel() since its always 1d array

?

Reply:
seems correct... rewrote that, more concise